presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -734,26 +734,63 @@ Procedures
 
 Use the CALL statement to perform data manipulation or administrative tasks. Procedures are available in the ``system`` schema of the catalog.
 
+CALL statement supports passing arguments by name or by position. Mixing position and named arguments is not supported.
+
+* Named arguments
+
+Procedure arguments all have a name. When passing arguments by name, arguments can be in any order
+and any optional argument can be omitted.
+
+    CALL iceberg.system.procedure_name(arg_name_2 => arg_2, arg_name_1 => arg_1);
```

Comment:
Formatting nit: suggest fixing the indent of this line and of line 751 so the vertical lines visible in a local doc build screenshot are not created. 

<img width="803" alt="Screenshot 2024-09-05 at 12 53 59 PM" src="https://github.com/user-attachments/assets/33557132-6368-4ba4-a095-f8e14eb48086">

Reply:
Thanks for pointing this out, fixed! Please take a look when available.